hello_world_on_chain/test/user.ts

```diff
@@ -93,22 +107,22 @@ describe("Base Sepolia Tests", function () {
         loadUserDeployedAddress(),
         loadUserContractABI(),
         signer
-    );
-
-    const evmLinkData = loadEVMLinkData("../inputs/out.json");
+    ) as UserContract;
 
-    it("Set signing key", async () => {
-        const publicKey = evmLinkData.publicKey;
-        const tx = await userContract.setTASigningKeyAddress(publicKey as any);
-        await tx.wait()
-    })
+    const evmLinkData: EVMLinkData = loadEVMLinkData("../inputs/out.json");
 
-    it("Verify TA", async () => {
-        const ta = evmLinkData.transitiveAttestation;
-        const tx = await userContract.verifyAttestedFnCallClaims(ta)
+    it("Verify TA", async (): Promise<void> => {
+        const publicKey: string = evmLinkData.publicKey;
+        const ta: string = evmLinkData.transitiveAttestation;
+        const tx: ethers.ContractTransactionResponse = await userContract.processTAHelloWorld(
+            publicKey as any,
```

Comment:
I tied removing these `as any` and I'm no longer seeing errors. Can you check that you can do that on your end as well?